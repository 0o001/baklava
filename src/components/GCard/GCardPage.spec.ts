import { shallowMount } from '@vue/test-utils';
import { GCardPage } from './';

describe('GCardPage', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallowMount(GCardPage);
  });

  it('should match snapshot when active is true', async () => {
    wrapper.setData({
      active: true,
    });

    await wrapper.vm.$nextTick();

    expect(wrapper.element).toMatchSnapshot();
  });

  it('should match snapshot when active is false', async () => {
    wrapper.setData({
      active: false,
    });

    await wrapper.vm.$nextTick();

    expect(wrapper.element).toMatchSnapshot();
  });
});
