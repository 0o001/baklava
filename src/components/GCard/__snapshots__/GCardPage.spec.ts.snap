// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GCardPage should match snapshot when active is false 1`] = `
<div
  style="display: none;"
/>
`;

exports[`GCardPage should match snapshot when active is true 1`] = `
<div
  style=""
/>
`;
